_integration-schemas/3plcentral/customers.md

```diff
@@ -0,0 +1,489 @@
+---
+tap: "3plcentral"
+version: "1.0"
+
+name: "customers"
+doc-link: "http://api.3plcentral.com/rels/"
+singer-schema: "https://github.com/singer-io/tap-3plcentral/blob/master/tap_3plcentral/schemas/customers.json"
+description: "This table contains information about customers."
+
+replication-method: "Full Table"
+
+api-method:
+    name: "REL documentation"
+    doc-link: "http://api.3plcentral.com/rels/customers/customers"
+attributes:
+  - name: "customer_id"
+    type: "integer"
+    primary-key: true
+    description: "The customer ID."
+    foreign-key-id: "customer-id"
+    
+  - name: "company_info"
+    type: "object"
+    description: "When identifying an existing contact, the lookup order is: 1) id, 2) code if no other fields present, 3) all fields (except id). If contact_id is not specified and other fields do not match an existing contact for the relevant customer, a new contact is created."
+    subattributes:
+      - name: "address1"
+        type: "string"
+        description: ""
+      - name: "address2"
+        type: "string"
+        description: ""
+      
+      - &address_status
+        name: "address_status"
+        type: "integer"
+        description: "The confirmation status. 0 - Unconfirmed, 1 - Confirmed, 2 - UserAccepted."
```

Comment:
Multiple values, so this should be a list:

```
The confirmation status. Possible values are:

- `0` - Unconfirmed
... etc
```

Remember to use backticks for literal values.